_data/willbarkoff/willbarkoff.md

```diff
@@ -0,0 +1,7 @@
+---
+name: William Barkoff
+institution: The Dalton School
+profile_pic: wbarkoff.png
```

Comment:
Image file path name seems incorrect, please change it to `willbarkoff.jpg`

Reply:
Sorry! I replaced the photo and forgot to replace the path name. I've fixed it now.